decidim-elections/app/helpers/decidim/elections/admin/steps_helper.rb

```diff
@@ -17,6 +17,21 @@ def steps(current_step)
             end
           end
         end
+
+        def fix_it_button_with_icon(link, icon_name, method = :get)
+          link_to(link, class: "button tiny button__secondary px-2 py-0", method:) do
+            "#{icon(icon_name, class: "fix-icon")} #{I18n.t("decidim.elections.admin.steps.create_election.errors.fix_it_text")}".html_safe
+          end
+        end
+
+        # i18n-tasks-use decidim.elections.admin.steps.create_election.technical_configuration.*
```

Comment:
I messed up the syntax (missing `t("XX")`), and also it seems like the i18n-task-use doesn't allow using the asterisk, so we need to fix the syntax and be explicit here:
```suggestion
        # i18n-tasks-use t("decidim.elections.admin.steps.create_election.technical_configuration.bulletin_board_server")
        # i18n-tasks-use t("decidim.elections.admin.steps.create_election.technical_configuration.authority_name")
        # i18n-tasks-use t("decidim.elections.admin.steps.create_election.technical_configuration.scheme_name")
```